Examples/Io/Root/include/ActsExamples/Io/Root/RootParticleWriter.hpp

```diff
@@ -82,17 +82,17 @@ class RootParticleWriter final : public WriterT<SimParticleContainer> {
   /// Production process type, i.e. what generated the particle.
   std::vector<uint32_t> m_process;
   /// Production position components in mm.
-  std::vector<float> m_vx;
-  std::vector<float> m_vy;
-  std::vector<float> m_vz;
```

Comment:
Any particular motivation for this?

Reply:
this was coming from https://github.com/acts-project/acts/pull/2241 I can revert that now